.circleci/config.yml

```diff
@@ -3,14 +3,10 @@ orbs:
   python: circleci/python@2.0.3
 
 definitions:
-  # Make sure we install the current project
-  # otherwise it looks like CircleCI wants to rely on using PYTHONPATH
-  # to find packages. But this errors due to our use of
-  # 'importlib.metadata'
   install-args: &install-args
       pip-dependency-file: requirements-dev.txt
-      pkg-manager: pip
-      args: "."
+      # `pip-dist` so we install the current project, and not just dependencies
+      pkg-manager: pip-dist
```

Comment:
`pip-dist` doesn't actually install with `pip-dependency-file` (just uses it as a cache key). So I guess I'd need `args: --requirement requrements-dev.txt` too :thinking: